eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -98,21 +97,49 @@ def fill_in_table_from_remapping(private_config_xml_file, release_version, refer
                 f"'{release_folder_name}') ON CONFLICT DO NOTHING")
             execute_query(pg_conn, query_insert)
 
-
-def fill_in_from_previous_inventory(private_config_xml_file, release_version):
-    query = ("select taxonomy_id, scientific_name, assembly, sources, total_num_variants, release_folder_name "
-            "from dbsnp_ensembl_species.release_species_inventory where sources='DBSNP - filesystem' and release_version=2")
+    query_origin_assembly_accession_info = (f"""select taxonomy, scientific_name, origin_assembly_accession, source, num_ss_ids
+        from eva_progress_tracker.remapping_tracker where release_version={release_version}""")
```

Comment:
I probably just missed something, but I don't get why we need the remapping source assemblies added to the clustering/release tracker this time?

Reply:
(Also, can this be refactored so the code is not duplicated?)

Reply:
Refactored.
There were some **origin_assemblies** in remapped_tracker which were not released last time (no entry in **clustering_release_tracker** release_version=**3**) and since we are only picking **assembly_accession** and not **origin_assembly** from remapping_tracker, these were not getting picked up there as well.  
Updated the code to include those.

Reply:
Right okay... in that case I think they need to be marked `should_be_clustered = False` so that they [aren't picked up for clustering](https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-clustering-automation/clustering_automation/cluster_from_mongo.py#L38) but are still picked up for release.

Reply:
done